mautic/mautic
# Allow use any form on preference center

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/6397
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Prefs page center wrap to form all source, from `<body>`
This PR remove it and add support to form  just wrap prefs slots/tokens.  This change allow use Unsubscribe feedback form /any forms on that page.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:

1. Create a form
2. Create a landing page, set it as the preference center. In the builder add some bloc form the pref center and add the form previously created with the token {form=ID}
3. Create an email, assign the pref center to this email.
4. Shoot the email
5. When you received the email, click on the unsubscribe link. You should land on the pref center.
6. Submit the form you integrated.
7. Check the form result and see that there is no submission.

#### Steps to test this PR:
1.  Repeat all steps
2. Both form should submit 

DOM HTML with 2 forms separately

![image](https://user-images.githubusercontent.com/462477/43465591-f3de21aa-94dd-11e8-80e8-7aa1fa3a1565.png)
